src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -135,7 +145,11 @@ export class ArtworkSidebarCurrentBidInfo extends React.Component<
               <Serif size="2" color="black60">
                 <br />
                 This auction has a{" "}
-                <Link onClick={this.onClickBuyerPremium.bind(this, mediator)}>
```

Comment:
In order to get tracking to fire i had to remove this binding. Is this now handled by babel?

Reply:
Heh, we're in weird JS territory right now 😄 

To poorly explain: `bind` does a couple things in JS -- it allows one to attach functions to different scopes (in the most common case, `this`) and it also serves as a sort of odd form of [partial application](https://towardsdatascience.com/javascript-currying-vs-partial-application-4db5b2442be8). Meaning, by calling `foo.bind(scope, someParam)` on a function you can "preload" the function with additional arguments (someParam) that can be plucked out of the second argument when foo is invoked:

```js
function MyComponent() {
  const foo = (name, location) => {
    console.log(name, location) // chris, washington
  }

  const handleClick = foo.bind(this, "washington") 

  return <div onClick={() => handleClick("chris") />
}
```

The old code `onClick={this.onClickBuyerPremium.bind(this, mediator)}>` was using this partial application form to pass `mediator` across before the actual click occurs, which will then pass the click event arguments. 

This is confusing, can hop on call to explain!

Reply:
Ah thanks! I was aware it was possible for that case but had never seen a practical application of it. I tried this in the console though and I think that the args are applied in order, so in this example "washington" would be bound to `name`:
```
>   const foo = (name, location) => {
     console.log(name, location) // chris, washington
   }

>   const handleClick = foo.bind(this, "washington")

> foo("a","b")
a b

> handleClick("A")
washington A
```